uint16 aout0
uint16 aout1
